taskweaver/role/role.py

```diff
@@ -62,6 +63,22 @@ def __init__(self, src: AppConfigSource) -> None:
             False,
         )
 
+        self.use_example = self._get_bool(
+            "use_example",
+            True,
+        )
+        self.example_base_path = self._get_path(
+            "example_base_path",
+            os.path.join(
+                self.src.app_base_path,
+                f"{self.name}_examples",
+            ),
+        )
```

Comment:
a bit wondering whether we should put all the examples in a common folder in project, i.e., `examples/{role_name}`. Also seems the role name has changed for code generator / code interperter, does that need to have any migration if reusing existing project folder with updated code?